### PR TITLE
WIP Tilt-Quadrotor

### PR DESCRIPTION
**Pull request by RicardoM17**
PR to add the Tilt-Quadrotor (new configuration) Airframe + Mixer files.

Not an expert with Github and PRs so please let me know if I'm doing something wrong.

### CODE REVIEW

File: ROMFS/px4fmu_common/mixers/tilt_quad.aux.mix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Tilt-Quadrotor mixer for PX4FMU (2/2) V2.0
+===========================
+
+This file defines the aux outputs mixer for a Tilt-quadrotor in the + configuration.
+
+
+# @output AUX1 servo motor 1
```
Comment (dagar) [inline ROMFS/px4fmu_common/mixers/tilt_quad.aux.mix]: Can you give these meaningful names? Aileron 1, 2, Elevator, etc.
https://docs.px4.io/en/airframes/airframe_reference.html#standard-vtol

File: src/examples/px4_simple_app/px4_simple_app.c [hunk 2]
```diff
@@ -38,92 +38,12 @@
...
-#include <uORB/uORB.h>
-#include <uORB/topics/sensor_combined.h>
-#include <uORB/topics/vehicle_attitude.h>
+#include <px4_log.h>
 
 __EXPORT int px4_simple_app_main(int argc, char *argv[]);
 
 int px4_simple_app_main(int argc, char *argv[])
 {
-	PX4_INFO("Hello Sky!");
```
Comment (dagar) [inline src/examples/px4_simple_app/px4_simple_app.c]: Unintentional change?

File: src/examples/renamed_px4_simple_app/px4_simple_app.c [hunk 3]
```diff
@@ -0,0 +1,129 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2012-2016 PX4 Development Team. All rights reserved.
+ *
```
Comment (dagar) [inline src/examples/renamed_px4_simple_app/px4_simple_app.c]: Unintentional change?

File: src/examples/px4_simple_app/px4_simple_app.c (hunk 2 above)
Comment (RicardoM17) [reply src/examples/px4_simple_app/px4_simple_app.c]: Yes probably. I only want to commit the Airframe file + the 2 mixer files. Anything else I've changed is irrelevant

File: ROMFS/px4fmu_common/mixers/tilt_quad.aux.mix (hunk 1 above)
Comment (RicardoM17) [reply ROMFS/px4fmu_common/mixers/tilt_quad.aux.mix]: I will change them for something more specific

**Comment by RicardoM17** [comment]
@dagar no more conflicts but I have a feeling this is still not right...

**Comment by dagar** [comment]
It looks like submodule changes have crept into your commit.

image

**Pull request closed without merging**